server/app/blocks.go

```diff
@@ -133,6 +133,36 @@ func (a *App) InsertBlocks(c store.Container, blocks []model.Block, modifiedByID
 	return blocks, nil
 }
 
+func (a *App) UpdateFileIDs(sourceBoardID string, blocks []model.Block) error {
```

Comment:
This isn't actually updating the fileIDs, it's copying files.  Perhaps a different name would be clearer.